web/profiles/custom/os2loop/themes/os2loop_theme/assets/app.scss

```diff
@@ -3,6 +3,10 @@
     background-color: #f2f2f2;
 }
 
+.bg-navbar-loop {
+    background-color: #41ab51;
```

Comment:
We should introduce variables for colors to make it easier to handle them in the near future.